README.md

```diff
@@ -107,19 +109,40 @@ github:1.11.3
 
 After this, just run `docker build -t my_jenkins_image -f Dockerfile`.
 
-Furthermore,  there are now 2 plugins of note included in this Jenkins Docker image that specifically facilitate the creation
-of CI/CD jobs and workflows for [OpenShift v3](https://github.com/openshift/origin):
+#### Installing using S2I build
+
+The [s2i](https://github.com/openshift/source-to-image) tool allows you to do additional modifications of this Jenkins image.
+For example, you can use S2I to copy custom Jenkins Jobs definitions, additional
+plugins or replace the default `config.xml` file with your own configuration.
+
+To do that, you can either use the standalone `s2i` tool, that will produce the
+customized Docker image or you can use OpenShift Source build strategy.
+
+In order to include your modifications in Jenkins image, you need to have a Git
+repository, with following directory structure:
+
+* `./plugins` folder that contains binary Jenkins plugins you want to copy into Jenkins 
+* `./plugins.txt` file that list the plugins you want to install (see the section above)
+* `./configuration/jobs` folder that contains the Jenkins job definitions
+* `./configuration/config.xml` file that contains your custom Jenkins configuration
```

Comment:
it's probably worth noting that if they do this, then the jenkins env variable JENKINS_PASSWORDD no longer applies (right?)